scripts/UploadBlobs.ps1

```diff
@@ -0,0 +1,142 @@
+<#
```

Comment:
Where does this copy over the artifacts from previous release?

Reply:
It doesn't. I'm still doing that manually.